eth-contracts/contracts/ServiceTypeManager.sol

```diff
@@ -1,11 +1,13 @@
 pragma solidity ^0.5.0;
 
+import "@openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol";
```

Comment:
should we remove from this contract if not used?

Reply:
yep! just pushed